test/e2e/page-objects/tribunals-case-api/getCaseData.js

```diff
@@ -0,0 +1,41 @@
+const config = require('config');
+
+const tribunalsApiUrl = config.get('api.url');
+const authCookie = '__auth-token';
+
+async function getMYACaseData(I, ccdCaseID) {
+  const res = await I.sendGetRequest(`${tribunalsApiUrl}/appeals?caseId=${ccdCaseID}`);
+
+  if (res && res.status === 200) {
+    if (res.data && res.data.appeal) {
+      return res.data.appeal;
+    }
+    throw Error('Invalid API Response)');
```

Comment:
'Invalid API Response appeal is missing from returned data' would be better - you always want to make whoever has to sift through log's life easier.